packages/react/README.md

```diff
@@ -33,19 +41,88 @@ Check out this [integration test](../spec/integration/react) as an example for h
 
 ##### `render(element[, options]): UniversalRenderImplementation`
 
-This is the main render method that you need to export from your entry file to render your application. It optionally accepts an options hash where you can configure certain runtime mixins.
+`render()` is `hops-react`'s main export. You are expected to call it in your applications main entry file and it is essentialy a shorthand: it creates and bootstraps a mixin container and calls its `render` method.
```

Comment:
I wonder whether it makes sense to import all the things in these code examples from `hops-react`, or whether we should rather point out, that all those are re-exported by the main package & have the examples sport imports from the main package accordingly.

Reply:
Hm. Good question. Maybe we should have another round through the documentation and update it accordingly, but I don't really want to do it in this PR, because this is just the backport to v12.x.